src/Main.cs

```diff
@@ -79,16 +92,15 @@ private void OnGUI() {
 					GUILayout.Box($"Player: {player.playerName}");
 				}
 				if (player.connectionToServer == null) {
-					if (GUILayout.Button("Kick")) {
-						player.GetComponent<NetworkIdentity>().connectionToClient.Disconnect();
-					}
+					if (GUILayout.Button("Kick"))
+						KickPlayer(player);
+					if (GUILayout.Button("Ban"))
+						BanPlayer(player);
```

Comment:
This should be covered by clang-format, please run it on your code
```suggestion
					if (GUILayout.Button("Kick")) {
						KickPlayer(player);
					}
					if (GUILayout.Button("Ban")) {
						BanPlayer(player);
					}
```